packages/opensrp-plans/README.md

```diff
@@ -0,0 +1,14 @@
+# Plans Views/Pages
```

Comment:
Does this doc need to be more detailed, or is that for later?

Reply:
Yes, Will update the doc once I add the missions/plan listview components and service hooks